src/main/java/org/mskcc/limsrest/util/IGOTools.java

```diff
@@ -1,7 +1,15 @@
 package org.mskcc.limsrest.util;
 
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
 public class IGOTools {
 
+    private static Set<String> EXCEPTIONALLY_VALID_IGO_IDS = new HashSet<String>(
+            Arrays.asList("POOLED_05605_AC_1")
```

Comment:
I don't know how sample receiving created this.  Since this is AmpliconSeq maybe it shouldn't even have been a CMO project and they will never look at the fastqs? Also instead of adding this exception we could change the LIMS and fastq names to the standard format?